README.md

```diff
@@ -47,16 +47,22 @@ The locks keyspace and table needs to be created before running any migrations.
 To apply all `.cql` files located in `/cql` in the classpath:
 
 ```java
+import com.datastax.driver.core.*;
+
 // Configure locking for coordination of multiple nodes
 CassandraLockConfig lockConfig = CassandraLockConfig.builder()
         .withTimeout(Duration.standardSeconds(3))
         .withPollingInterval(Duration.millis(500))
-        .build())
+        .build();
+
+// Create a Cassandra session
+Cluster cluster = Cluster.builder.addContactPoint("localhost").build;
```

Comment:
Build needs brackets after

Reply:
True, I copied from Scala and missed that